app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -1131,4 +1163,4 @@
     ],
     "path": "app/javascript/mastodon/features/ui/components/video_modal.json"
   }
-]
\ No newline at end of file
+]
```

Comment:
I did not, that's very convenient. It should probably be added to the development docs.